snc.sh

```diff
@@ -17,7 +17,6 @@ if [[ ${OKD_VERSION} != "none" ]]
 then
     OPENSHIFT_VERSION=${OKD_VERSION}
     MIRROR=${MIRROR:-https://github.com/openshift/okd/releases/download}
-    OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE:-quay.io/openshift/okd:${OPENSHIFT_VERSION}}
```

Comment:
If you remove this one then https://github.com/code-ready/snc/blob/master/snc.sh#L305 will be set as the image override, not something we want in case of OKD.

Reply:
Okay so looks like now https://github.com/openshift/okd/releases/download/4.6.0-0.okd-2020-11-27-200126/release.txt it have release txt which we can extract release info also